src/averell/readers/ecpa.py

```diff
@@ -0,0 +1,142 @@
+import json
+import re
+import xml.etree.ElementTree as ETree
+
+from averell.utils import TEI_NAMESPACE as NS
+from averell.utils import XML_NS
+
+ECEP_NS = "{http://www.eighteenthcenturypoetry.org/ns}"
+
+
+def get_poem_info(xml_file, lines_info, authors):
+    """Poem parser for 'ECPA corpus'.
+    We read the data and find elements like title, author, year, etc. Then
+    we iterate over the poem text and we look for each stanza, line, word
+    and syllable data.
+
+    :param xml_file: Path for the poem xml file
+    :param lines_info: Path for the lines json file
+    :param authors: dict with authors info
+    :return: Dict with the data obtained from the poem
+    :rtype: dict
+    """
+    poem = {}
+    tree = ETree.parse(xml_file)
+    root = tree.getroot()
+    manually_checked = False
+    metadata = root.attrib
+    title = root.find(f".//{NS}head[@type='main']")
+    poem_id = metadata.get(f"{XML_NS}id")
+    poem_info = authors[1].get(poem_id)
+    if poem_info:
+        title_text = poem_info.get("title")
+    else:
+        title_text = " ".join(word.text for word in title.findall(f"{NS}w"))
+    author = root.find(f"{NS}link[@type='author']").get("target").split("#")[1]
+    try:
+        author_name = next(aut.get("name") for aut in authors[0].values() if
+                           aut.get("author") == author)
+    except StopIteration:
+        author_name = '-'
```

Comment:
We need to come up with a solution in Averell for missing data. Is it going to be `"-"`, `""`, `None`, `"N/A"`?